theano/sandbox/gpuarray/__init__.py

```diff
@@ -64,8 +65,25 @@ def init_dev(dev, name=None):
     reg_context(name, context)
     pygpu_activated = True
     if config.print_active_device:
-        print("Mapped name %s to device %s: %s" % (name, dev, context.devname),
+        warn = None
+        cudnn_version = ""
+        if dev.startswith('cuda'):
+            cudnn_version = " (CuDNN not available)"
+            try:
+                cudnn_version = dnn.version()
+                # 4100 should not print warning with cudnn 4 final.
+                if cudnn_version > 4100:
```

Comment:
How about `>= 5000`?  But otherwise I agree that 4100 is probably safe if NVIDIA follows previous numbering standards.